### ISSUE TITLE
@hono/qwik-city on Cloudflare Pages?

### ISSUE THREAD

**Issue by dolsem** [honojs/middleware#406]
Is it possible to use @hono/qwik-city in a Cloudflare Pages project? Does it work with @hono/vite-cloudflare-pages? I feel like I've tried everything, but I cannot figure out how to make these two plugins work with each other.

**Comment by yusukebe** [comment]
Hi @dolsem 

I'm sure you can do it, but `@hono/qwik-city` is not complete and I cannot support it.

**Comment by dolsem** [comment]
Do you mean you cannot support `@hono/qwik-city` or just the Cloudflare Pages integration? If you're planning to finish `@hono/qwik-city`, how long do you think it will take? I believe there is currently no good way to combine Qwik with API routes on Cloudflare Pages, so being able to do it with Hono would provide a lot of value

**Comment by yusukebe** [comment]
> Do you mean you cannot support `@hono/qwik-city` or just the Cloudflare Pages integration?

"Cloudflare Paages Integration". But, actually, I don't have time to make `@hono/qwik-city` and the priority is low. So, I want someone's help!

**Comment by PatrickJS** [comment]
@yusukebe I can update it. we're using the code as a starting point for our Prod app (only a few things needed to be updated in the code)

**Comment by yusukebe** [comment]
Hi @PatrickJS 

Great. Can you create a PR (though it's good you'll wait for announcements from Cloudflare today!)?

**Comment by PatrickJS** [comment]
yeah, I updated the qwik middleware you made. I think it's better if we want for the vite 6 "environments" api changes to get into qwik then it will be even simpler

**Comment by yusukebe** [comment]
@PatrickJS 

Ideally, it's great that we will have a Vite plugin for developing Qwik with Hono and adapters for Cloudflare Pages, etc. It's like "hono-remix-adapter" recently I made: https://github.com/yusukebe/hono-remix-adapter

I've struggled for a few hours to find a good solution, but I can't find it.

I am waiting for your idea!
